app/views/shared/schools/mentors/_can_not_remove.html.erb

```diff
@@ -0,0 +1,9 @@
+<%# locals: (mentor:) -%>
+<label class="govuk-label govuk-label--l">
+  <span class="govuk-caption-l"><%= mentor.full_name %></span>
```

Comment:
When rendered in the support console, this caption should include the name of the school, too.

e.g. like this:

https://github.com/DFE-Digital/itt-mentor-services/blob/009c48f771bf9ebf737fca665b430617facc7f48/app/views/placements/support/schools/mentors/remove.html.erb#L12